Week-2/birthdayMessage/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Happy Birthday Message Creator</title>
+    <style>
```

Comment:
It’s good practice to separate styles into an external stylesheet, especially as your project grows. This keeps your HTML file cleaner and promotes reusability of styles across different parts of your website.

Reply:
```
<a href="https://www.metallica.com/" target="_blank" rel="noopener noreferrer">official website</a>

```